src/seedu/addressbook/AddressBook.java

```diff
@@ -482,10 +482,10 @@ private static String getMessageForPersonsDisplayedSummary(ArrayList<String[]> p
      * @param keywords for searching
      * @return list of persons in full model with name containing some of the keywords
      */
-    private static ArrayList<String[]> getPersonsWithNameContainingAnyKeyword(Collection<String> keywords) {
+    private static ArrayList<String[]> getPersonsWithNameContainingAnyKeywordCaseInsensitive(Collection<String> keywords) {
         final ArrayList<String[]> matchedPersons = new ArrayList<>();
         for (String[] person : getAllPersonsInAddressBook()) {
-            final Set<String> wordsInName = new HashSet<>(splitByWhitespace(getNameFromPerson(person)));
+            final Set<String> wordsInName = new HashSet<>(splitByWhitespace(getNameFromPerson(person).toLowerCase()));
```

Comment:
please update the tests to reflect this change?
also ensure that this change doesn't affect the way data is stored internally.